panel/src/components/Views/Login/index.js

```diff
@@ -0,0 +1,28 @@
+import LoginView from "./LoginView.vue";
+
+import LoginForm from "./LoginForm.vue";
+import LoginAlert from "./LoginAlert.vue";
+import LoginCodeForm from "./LoginCodeForm.vue";
+
+import InstallationView from "./InstallationView.vue";
+import ResetPasswordView from "./ResetPasswordView.vue";
+
+import UserInfo from "./UserInfo.vue";
```

Comment:
Would it make sense to rename this into LoginUserInfo as well while we are already at it?

Reply:
Sine ResetPasswordView is also using it and it could be used in other user-related contexts as well, I moved it to the Users folder and kept the name as is.